content/capture-app/app-android-settings-configuration.md

```diff
@@ -256,14 +256,24 @@ It allows the admin user to enable or disable the option to show the filters rel
 
 ### Program { #capture_app_android_settings_webapp_appearance_program }
 
-It allows the admin user to specify which filters should be shown and enable or disable the option to show the completion percentage of the data entry form.
+It allows the admin user to specify which filters should be shown.
 
 #### Global settings { #capture_app_android_settings_webapp_appearance_program_global }
 
 Global settings apply to all programs that an android user has access to.
 
 ![](resources/images/capture-app-appearance-program-global.png){width=75%}
 
+Percentage (%) complete in Program: 
+Enable or disable the option to show the completion percentage of the data entry form.
+
+TEI referrals (**New 2.3.1**):	
+It allows to switch off/on the TEI referral option.
+
+Collapse sections in form (**New 2.3.1**):  
+Current behavior lets the sections collapse and expand in an accordion style. This option allows overriding or changing this behavior.	
```

Comment:
Add:

This option overrides the behavior by keeping all the sections of the form (registration and events) open and removing the "next" button in each section